### DISCUSSION TITLE
빵빠레 울렸는데 2번이나 실패했어요

### DISCUSSION THREAD

**Discussion by yyyy0079** [changdoc/naver-vaccine-macro#88] [Q&A]
크롬으로  한병원 지정해서...1아이디, 1개의 창만 띄워놓고...계속 시도했는데
빵빠레 울렸는데 실패네요.
신청확인이 마지막에 안눌러진건지 뭐가 잘못된건지 모르겠어요.
아님 그만큼 늦어서 그런걸까요?
두번이나 빵빠레 듣고 실패하니 좋다가 급다운..ㅠㅠ

**Comment by changdoc** [comment]
빵빠레는 성공 여부에 관계 없이 예약 시도 후 결과 페이지에 접근하게 되면 알려주는 알람 용도로 설정되어 있긴합니다.

다음 버전에는 
시도나 실패때는 다른 사운드가 재생되도록 하고
성공으로 확인될때만 재생되도록 수정해두겠습니다.

**Reply by yyyy0079** [reply]
아~다 울리는거군요...내일 다시 시도해봐야겠어요...방법 올려주셔서 너무 감사합니다.

**Reply by seokwonchoisnu** [reply]
간격이 2초이후로 패치된부터는 성공이 잘안되나요? 저도 오늘 아침부터 여기저기 다해봤는데 빵빠레 울리고나서 접근했는데 2번접근 되었었는데 다 실패했네요 ㅠㅜ..

**Reply by gingno** [reply]
저 2초 패치 되고 시도 계속 했는데 오늘 성공 했습니다. ^^

**Reply by seokwonchoisnu** [reply]
2일정도 하셨나요?

**Reply by seokwonchoisnu** [reply]
> 간격이 2초이후로 패치된부터는 성공이 잘안되나요? 저도 오늘 아침부터 여기저기 다해봤는데 빵빠레 울리고나서 접근했는데 2번접근 되었었는데 다 실패했네요 ㅠㅜ..

3일 정도 6번 트라이만에 1시까지인 병원 12시 50분에 성공했습니다. 정말 감사합니다.
